ui/imports/shared/views/AssetsView.qml

```diff
@@ -251,27 +293,29 @@ ColumnLayout {
         id: tokenDelegate
         TokenDelegate {
             objectName: "AssetView_TokenListItem_" + (!!modelData ? modelData.symbol : "")
-            readonly property string balance: !!modelData ? "%1".arg(modelData.enabledNetworkBalance.amount) : "" // Needed for the tests
+            readonly property string balance: !!modelData && !!modelData.currentBalance? "%1".arg(modelData.currentBalance) : "" // Needed for the tests
             errorTooltipText_1: !!modelData && !!networkConnectionStore ? networkConnectionStore.getBlockchainNetworkDownTextForToken(modelData.balances) : ""
             errorTooltipText_2: !!networkConnectionStore ? networkConnectionStore.getMarketNetworkDownText() : ""
             subTitle: {
-                if (!modelData) {
+                if (!modelData || !modelData.symbol) {
                     return ""
                 }
                 if (networkConnectionStore && networkConnectionStore.noTokenBalanceAvailable) {
                     return ""
                 }
-                return "%1 %2".arg(LocaleUtils.stripTrailingZeroes(LocaleUtils.numberToLocaleString(modelData.enabledNetworkBalance.amount, 6), Qt.locale()))
-                              .arg(modelData.enabledNetworkBalance.symbol)
+                return LocaleUtils.currencyAmountToLocaleString(root.currencyStore.getCurrencyAmount(modelData.currentBalance, modelData.symbol))
+            }
+            currencyBalance.text: {
+                let totalCurrencyBalance = modelData && modelData.currentCurrencyBalance ? modelData.currentCurrencyBalance : 0
+                return LocaleUtils.currencyAmountToLocaleString(root.currencyStore.getCurrentCurrencyAmount(totalCurrencyBalance))
             }
             errorMode: !!networkConnectionStore ? networkConnectionStore.noBlockchainConnectionAndNoCache && !networkConnectionStore.noMarketConnectionAndNoCache : false
             errorIcon.tooltip.text: !!networkConnectionStore ? networkConnectionStore.noBlockchainConnectionAndNoCacheText : ""
             onClicked: (itemId, mouse) => {
                            if (mouse.button === Qt.LeftButton) {
-                               RootStore.getHistoricalDataForToken(modelData.symbol, RootStore.currencyStore.currentCurrency)
+                               RootStore.getHistoricalDataForToken(modelData.symbol, root.currencyStore.currentCurrency)
                                d.selectedAssetIndex = delegateIndex
-                               let selectedView = !!modelData.communityId ? communityAssetsLV : regularAssetsLV
```

Comment:
Why did you revert this?

Reply:
Hey @caybro Iv basically updated the logic here from having 2 listviews to have a single listview with sections instead. Its no reverted simply updated . Same for all the comments with regards to these changes